src/stories/Overlay.stories.tsx

```diff
@@ -311,7 +311,7 @@ export const NestedOverlays = () => {
                   Create a list to organize your starred repositories.
                 </Text>
                 <TextInput placeholder="Name this list" sx={{mb: 2}} />
-                <TextInput as="textarea" placeholder="Write a description" rows="3" sx={{mb: 2, textarea: {p: 2}}} />
+                <TextInput as="textarea" placeholder="Write a description" rows={3} sx={{mb: 2, textarea: {p: 2}}} />
```

Comment:
Could we just change this to a Textarea?

Reply:
Yes, but until we deprecate the `as` prop for TextInput I think consumers will still do this. If people do this in their apps, I like that we are type-checking this use-case in our codebase.

Reply:
Ah ok. Let's keep it as-is then.